docs/admins/assignskill.rst

```diff
@@ -0,0 +1,158 @@
+                    :scale: 30 %
+
+2. To create a new subtopic, click on **+ Add Subtopic**, enter a title and click |createsubtopic|.
+
+.. |createsubtopic| image:: /images/create_subtopic.png
+                    :scale: 40 %
+
+3. To assign skills to a subtopic, drag the skill from the list of **Uncategorized Skills** on the right side and drop it under the relevant subtopic. To remove a skill from the subtopic, drag the skill back to the Uncategorized Skills list.
+
+.. figure:: /images/subtopics_page.png
+   :alt: Creating a subtopic
+   :scale: 30 %
+
+   *Fig. 5* 
+
+4. To unassign a skill from a topic, click the |trashcan| button next to the skill in the **Uncategorized Skills** list. This will not delete the skill, but move it back to the **Unused Skills** tab in the Topics and Skills dashboard.
+
+.. |trashcan| image:: /images/trashcan.png
+              :alt: Delete skill button
+              :scale: 40 %
+
+5. To delete a subtopic, click the red **X** next to the Subtopic Title. This moves any associated skills back to the **Uncategorized Skills** list.
+
+6. To create a high-level description for the subtopic that learners will see in the practice session, click the |pencil| button next to the Subtopic Title. Enter the description in the text box under **Subtopic Page**. This will help learners decide which subtopics to practice.
+
+.. |pencil| image:: /images/pencil.png
+            :scale: 40 %
+
+.. important::
+
+   Ensure that each subtopic contains a unique list of skills, i.e., skills are not repeated in different subtopics. 
+
+7. When you have finished creating subtopics and assigning skills to them, click |savechanges|.
+
+.. |savechanges| image:: /images/save_changes.png
+                 :alt: Save Changes button
+                 :scale: 40 %
+
+.. _createquestion:
+
+Creating a question for a skill
+--------------------------------
+
+Questions can be created either through the Topic Editor or submitted by contributors. 
+
+1. To create a question from the Topic Editor, click on the topic, and then the **Questions Editor** |questioneditor| button in the top-right corner.
+
+.. |questioneditor| image:: /images/question_editor.png
+                    :scale: 30 %
+
+2. Click the **Create Question** button.
+
+3. Select the skill for which you want to create the question and click **Next**.
+
+.. figure:: /images/select_skill_question.png
+   :alt: Selecting skills for questions
+   :scale: 30 %
+
+   *Fig. 6*
+
+4. Select the question's level of difficulty—Easy, Medium or Hard—and click **Done**. If there were :ref:`rubrics <rubrics>` written for these levels when the skill was created, you will see them here.
+
+5. The **Question Editor** box opens. This works just like an :ref:`Exploration Editor <createcontent>` where you can create questions, add interactions, responses and feedback. You also have to add a hint in order to publish the question. When you have finished creating the question, click |publishquestion|.
```

Comment:
Here, for interactions that have solutions (eg: TextInput has it, but multiple choice does not), that also has to be specified.

Reply:
Done